src/ucp/proto/proto_am.c

```diff
@@ -23,12 +23,13 @@ static size_t ucp_proto_pack(void *dest, void *arg)
     return 0;
 }
 
-ucs_status_t ucp_proto_progress_am_bcopy_single(uct_pending_req_t *self)
+ucs_status_t ucp_proto_progress_am_bcopy_single(uct_pending_req_t *self,
+                                                ucs_status_t status)
```

Comment:
In UCT the uct_pending_callback_t prototype has been added the status argument for the case where during the time since posting the request an error has occured and progressing it is no longer possible.